src/Dhall/YAML.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE DeriveDataTypeable #-}
```

Comment:
I don't think you need this module any longer.  All it does is re-export `codeToValue` from the `Dhall.JSON` module

Reply:
Right, removed it.